tools/test/audio/std_utils/level_dbfs.m

```diff
@@ -9,38 +9,12 @@
 % dbfs - sigmal level in dBFS
 %
 
-%%
-% Copyright (c) 2017, Intel Corporation
-% All rights reserved.
-%
-% Redistribution and use in source and binary forms, with or without
-% modification, are permitted provided that the following conditions are met:
-%   * Redistributions of source code must retain the above copyright
-%     notice, this list of conditions and the following disclaimer.
-%   * Redistributions in binary form must reproduce the above copyright
-%     notice, this list of conditions and the following disclaimer in the
-%     documentation and/or other materials provided with the distribution.
-%   * Neither the name of the Intel Corporation nor the
-%     names of its contributors may be used to endorse or promote products
-%     derived from this software without specific prior written permission.
-%
-% THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
-% AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-% IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
-% ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
-% LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
-% CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
-% SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
-% INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
-% CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
-% ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
-% POSSIBILITY OF SUCH DAMAGE.
-%
+% SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Not sure if Checkpatch can even recognize this line as it's not the first one?

Reply:
I don't want to place the copyright as first line since then Matlab/Octave would show it as help text for the function (command "help level_dbfs"). The copyright is not relevant to see every time when need to check the needed inputs/outputs for the function. The empty line between the comments helps to omit the copyright also from the end of help text.

Reply:
Fair enough I guess. I really got rusty as I haven't used Matlab/Octave anytime recent and the only time I did use it it wasn't that serious anyway.

Reply:
Yep. When time and interest we could find out if there are code style conventions to utilize. So far I've used mostly emacs defaults for Octave mode. But it would not hurt to have e.g. "a = b + c" instead of "a = b+c" like we do in C.